### DISCUSSION TITLE
Power App-Code Review-ArchitectureReview-1-0-0-9 won't import

### DISCUSSION THREAD

**Discussion by unlimitedduck** [microsoft/powerapps-tools#891] [Q&A]
I have tried since I seen it posted on Twitter of the App created by Mehdi Slaoui Andaloussi to import it via the zip file supplied on GitHub.  However, each time I have tried it keeps giving me a red banner with the following: "Something went wrong. Please try again later."

I'm hoping @slaouist you are able to fill in maybe what I'm missing to import this directly into Power Apps.  
Link in question:  https://github.com/microsoft/powerapps-tools/blob/master/Tools/Apps/Microsoft.PowerApps.CodeReview/ArchitectureReview_1_0_0_9.zip

Is there something that needs to be turned on before this can be recognized when importing?  Appreciate any insight or help you can give.

Thank you.

**Comment by knoxknocks** [comment]
I'm having the same issue...were you able to resolve the import issue? Thanks!

**Reply by ilias-pazas** [reply]
Same problem here....

**Reply by unlimitedduck** [reply]
@knoxknocks and @ilias-pazas have not heard from the creator yet.  Not sure if the App includes connectors not mentioned that are premium or not.

I did try to do import as a solution and from that got further along, but seemed to have CDS connections.  If CDS is truly required to utilize this I don't believe it was mentioned in the blog nor video unless I missed it.  It just didn't scream out if it was mentioned.

@slaouist pinging you again for others having same issue and we all would like some guidance on your solution.

**Reply by ilias-pazas** [reply]
Power Platform is evolving fast and we are trying to adopt, but also integrate tooling in the daily development process. It is critical that tools behave as expected and it will be great to get some feedback and provide help to younger practitioners.

**Reply by unlimitedduck** [reply]
@knoxknocks and @ilias-pazas I heard back from @slaouist via Twitter.  

1.  This is not a Canvas App but a Power App Solutions.  Means must be loaded via the Solutions section in Power Apps
2.  You must have premium or permissions to the CDS portion of your tenant in question for the Solution utilized CDS connectors.

My end that's a bummer for don't have premium access for CDS and thus back to logging everything for the builds in OneNote to locate where things were tied to where.

**Reply by knoxknocks** [reply]
Thanks for the update! Bummer we don't have CDS either
